src/acvp_lcl.h

```diff
@@ -283,6 +285,7 @@
  * https://github.com/usnistgov/ACVP/blob/master/artifacts/acvp_sub_kdf135_ikev2.txt
  */
 #define ACVP_KDF135_IKEV2_SPI_STR_MAX 32
+#define ACVP_KDF135_IKEV2_SPI_BYTE_MAX ACVP_KDF135_IKEV2_SPI_STR_MAX/2
```

Comment:
Please surround macro value with parentheses to keep order of operations clear to compiler